cleanlab_studio/internal/tlm/validation.py

```diff
@@ -17,6 +17,36 @@
 )
 
 
+def validate_tlm_prompt_kwargs_constrain_outputs(
+    prompt: Union[str, Sequence[str]],
+    **kwargs: Any,
+) -> Union[Optional[List[str]], Optional[List[Optional[List[str]]]]]:
+    # validate kwargs - only allow constrain_outputs
+    supported_kwargs = {"constrain_outputs"}
```

Comment:
[nit] define this as constant here: https://github.com/cleanlab/cleanlab-studio/blob/main/cleanlab_studio/internal/constants.py ?